resources/assets/js/components/JobBuilder/Review/JobReview.tsx

```diff
@@ -860,7 +861,7 @@ export const JobReview: React.FunctionComponent<JobReviewProps &
             >
               <FormattedMessage
                 id="jobBuilder.review.button.submit"
-                defaultMessage="Looks good!"
+                defaultMessage="Save and Return to Summary"
                 description="Label of Job Review Submission Button"
```

Comment:
Last thing to do is changing this button into an link (which can look like a button). Also, add a comment saying the Congrats modal is will be moved to the summary page.

Reply:
@gggrant I already have it open so I can just pop it in for you.

Reply:
Awesome, thanks!